### PR TITLE
Improve performance by pre-encoding JSON property names in serializer

### LINKED ISSUE #8782: Pre-encode JSON property names
Can improve performance by pre-encoding JSON property names in serializer.

See KeyVault CertificateBackup.cs as an example.

https://github.com/Azure/azure-sdk-for-net/pull/8677#discussion_r345401754

### PR DESCRIPTION
**Pull request by Marusyk**
Pre-encode JSON property names

Fixes #8782

Please review
Thank you in advance

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs [hunk 1]
```diff
@@ -18,6 +18,11 @@ internal static class TextAnalyticsServiceSerializer
 
         #region Serialize Inputs
 
+        private static readonly JsonEncodedText s_countryHint = JsonEncodedText.Encode("countryHint");
```
Comment (maririos) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs]: could we replace `static readonly` with `const` ?

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs [hunk 2]
```diff
@@ -18,6 +18,11 @@ internal static class TextAnalyticsServiceSerializer
...
+        private static readonly JsonEncodedText s_id = JsonEncodedText.Encode("id");
+        private static readonly JsonEncodedText s_text = JsonEncodedText.Encode("text");
+        private static readonly JsonEncodedText s_language = JsonEncodedText.Encode("language");
+
```
Comment (maririos) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs]: could we include `documents` too?

**Comment by maririos** [comment]
Thanks @Marusyk !
Just wondering, I see you did it for serializing the input. Would it be ok to do it for the deserialization  part?

**Review by heaths — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs (hunk 1 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs]: Nevermind. won't work

**Review by maririos — APPROVED** [review]

**Comment by maririos** [comment]
Thank you @Marusyk for your contribution!

**Pull request merged**